compiler-rt/lib/scudo/standalone/secondary.h

```diff
@@ -314,16 +328,22 @@ class MapAllocatorCache {
 
       // All excess entries are evicted from the cache
       while (needToEvict()) {
+        EntryListT EvictionListType;
```

Comment:
It's probably worth a comment saying why you are evicting from the decommitted list first.